### ISSUE TITLE
Graphe the timing spent on each 3rd party server

### ISSUE THREAD

**Issue by jpvincent** [sitespeedio/grafana-bootstrap-docker#2]
On the old demo dashboard, we had a nice graph showing the slowest assets, over time.
![image](https://cloud.githubusercontent.com/assets/427625/19728296/2cc75054-9b93-11e6-8c1d-e19bbc323f10.png)

It was nice because it could be used to look at the availability and performance of the 3rd party (and some slow 1st party resource).
We could produce at least for the public demo a graph showing the response time per domain (it will probably store less data). It wont tell much about 1st party resources having troubles, but in my case it will be enough to monitor how 3rd parties scripts are performing.

**Comment by soulgalore** [comment]
Yes I see your point @jpvincent . One of the main issues with 4.0 was that it generated too much data by default into Graphite. Let us update the documentation to how to send the metrics as first step and then maybe setup one more dashboard (or include in the current 3rd party dashboard).

**Comment by gajus** [comment]
Any update on this?

**Comment by jpvincent** [comment]
hi @gajus 
I ended up implementing it by creating a dashboard and enabling more metrics to be sent to graphite, do you need a code example ?
I can confirm it does take a lot of storage space though, be careful
